BinaryData/Resource/TranslationKeysExtractor.cmake

```diff
@@ -0,0 +1,31 @@
+file(GLOB_RECURSE TranslationSourceFiles
+    ${CMAKE_CURRENT_SOURCE_DIR}/Source/*.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/Source/*.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/Dependencies/Misc/*.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/Dependencies/Misc/*.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/JUCE/modules/juce_audio_devices/*.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/JUCE/modules/juce_audio_devices/*.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/JUCE/modules/juce_gui_basics/*.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/JUCE/modules/juce_gui_basics/*.h
+)
+
+set(TRANSLATION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/translation_keys.txt")
```

Comment:
CMAKE_CURRENT_SOURCE_DIR is not a good path because git tracks it. You can specify the path with an argument in CMakeLists.txt and use a path relative to CMAKE_CURRENT_BINARY_DIR. The name of the file can be in camel case for consistency.